rpcd/playbooks/ironic-pre-config.yml

```diff
@@ -0,0 +1,117 @@
+      config_template:
+        src: "../../openstack-ansible/etc/openstack_deploy/openstack_user_config.yml.aio"
+        dest: "../../openstack-ansible/etc/openstack_deploy/openstack_user_config.yml.aio"
+        owner: "root"
+        group: "root"
+        mode: "0644"
+        config_overrides: "{{ cidr_networks_ironic }}"
+        config_type: "yaml"
+        list_extend: true
+
+    - name: Add IPs to used_ips list
+      config_template:
+        src: "../../openstack-ansible/etc/openstack_deploy/openstack_user_config.yml.aio"
+        dest: "../../openstack-ansible/etc/openstack_deploy/openstack_user_config.yml.aio"
+        owner: "root"
+        group: "root"
+        mode: "0644"
+        config_overrides: "{{ used_ips_ironic }}"
+        config_type: "yaml"
+        list_extend: true
+
+    - name: Setup global overrides
+      config_template:
+        src: "../../openstack-ansible/etc/openstack_deploy/openstack_user_config.yml.aio"
+        dest: "../../openstack-ansible/etc/openstack_deploy/openstack_user_config.yml.aio"
+        owner: "root"
+        group: "root"
+        mode: "0644"
+        config_overrides: "{{ ironic_networks }}"
+        config_type: "yaml"
+        list_extend: true
+
+    - name: Setup endpoints for swift/ironic health checks
+      config_template:
+       src: "../etc/openstack_deploy/user_osa_variables_overrides.yml"
+       dest: "../etc/openstack_deploy/user_osa_variables_overrides.yml"
+       owner: "root"
+       group: "root"
+       mode: "0644"
+       config_overrides: "{{ ironic_endpoint_config }}"
+       config_type: "yaml"
+
+    - name: Configure neutron DHCP to be non-authorative
+      config_template:
+        src: "../etc/openstack_deploy/user_osa_variables_overrides.yml"
+        dest: "../etc/openstack_deploy/user_osa_variables_overrides.yml"
+        owner: "root"
+        group: "root"
+        mode: "0644"
+        config_overrides: "{{ ironic_neutron_dhcp_config }}"
+        config_type: "yaml"
+
+  vars:
+    cidr_networks_ironic:
+      cidr_networks:
+           tftp: "172.29.232.0/22"
+           ironic-ipmi: "172.29.228.0/22"
+    ironic_networks:
+      global_overrides:
+        provider_networks:
+          - network:
+              container_bridge: "br-ironic-ipmi"
+              container_type: "veth"
+              container_interface: "eth_ipmi"
+              ip_from_q: "ironic-ipmi"
+              type: "raw"
+              group_binds:
+              - ironic-infra_hosts
+          - network:
+              container_bridge: "br-tftp"
+              container_type: "veth"
+              container_interface: "eth_tftp"
+              ip_from_q: "ironic-ipmi"
+              type: "flat"
+              net_name: "tftp"
+              ip_from_q: "tftp"
+              group_binds:
+              - neutron_linuxbridge_agent
+              - ironic_all
+    ironic_endpoint_config:
+      extra_lb_vip_addresses:
+        - "{{ ansible_eth0.ipv4.address }}"
```

Comment:
Changing this to external_lb_vip_address fixes the RepresenterError. This might be an issue with either the config_template or ansible.